teps/0110-decouple-catalog-organization-and-reference.md

```diff
@@ -239,7 +239,7 @@ this ordering of work, the Official Catalog will have the Git-Based organization
 [tep-0079]: ./0079-tekton-catalog-support-tiers.md
 [hub]: https://hub.tekton.dev
 [catalog]: https://github.com/tektoncd/catalog
-[hub-resolver]: https://github.com/sbwsg/hubresolver/tree/42962892535f19e9f1f9cd3457f567dd121d57ec
+[hub-resolver]:
```

Comment:
Can we update it to https://github.com/tektoncd/resolution/tree/main/hubresolver instead of leaving a broken link?

Reply:
Oops, done, thanks!